citizen.ts

```diff
@@ -44,8 +45,27 @@ export const issueFirstClassCitizenship = async (
   const sortedContractsByEpoch =
     groupContractsByEpochFromSimnetPlan(simnetPlan);
 
+  const simnetAddresses = [...simnet.getAccounts().values()];
+
+  const balancesMap = new Map(
+    Array.from(simnetAddresses, (address) => {
+      try {
+        const balanceHex = simnet.runSnippet(`(stx-get-balance '${address})`);
+        return [address, cvToValue(hexToCV(balanceHex))];
+      } catch (error) {
+        return [address, BigInt(0)];
```

Comment:
I'm not sure we should catch this. It should propagate up. Can we do anything meaningful with a tuple of `address, BigInt(0)`?